mmv1/third_party/terraform/services/netapp/resource_netapp_volume_test.go

```diff
@@ -273,14 +289,66 @@ data "google_compute_network" "default" {
 func testAccNetappVolume_volumeBasicExample_updatesnapshot(context map[string]interface{}) string {
 	return acctest.Nprintf(`
 
-resource "google_netapp_storage_pool" "default" {
-	name = "tf-test-test-pool%{random_suffix}"
+resource "google_netapp_storage_pool" "default2" {
```

Comment:
nit: we use spaces instead of tabs for spacing in the resource configurations. Would you mind fixing all the resources block in this file. Thanks!

Reply:
Oh, I didn't realise. Fixed.